src/card/card-expandable.jsx

```diff
@@ -57,54 +61,23 @@ const CardExpandable = React.createClass({
     };
   },
 
-  //to update theme inside state whenever a new theme is passed down
-  //from the parent / owner using context
   componentWillReceiveProps(nextProps, nextContext) {
-    let newMuiTheme = nextContext.muiTheme ? nextContext.muiTheme : this.state.muiTheme;
-    this.setState({muiTheme: newMuiTheme});
-  },
-
-  getStyles() {
-    const contextKeys = this.constructor.getRelevantContextKeys(this.state.muiTheme);
-
-    const directionStyle = contextKeys.isRtl ? {
```

Comment:
Yeah I tested this and realized it was a bug because it was reversing the `rtl` logic :grin: